inf_post/include/checks.h

```diff
@@ -0,0 +1,14 @@
+#ifndef CHECKS_H
+#define CHECKS_h
```

Comment:
defined var and the upper check's  differ

Reply:
+